### PR TITLE
Create ejbcontainer.bindings_fat and port default bindings test

### PR DESCRIPTION
**Pull request by olendvcook**
Created a new project for bindings tests and ported default bindings test.  Failing tests currently aren't ran because the TestClass code that adds the servlets is commented out.

Added Java:global lookup tests as a sanity check that the application is correctly installed and current bindings work.

### CODE REVIEW

**Comment by olendvcook** [comment]
#build

File: dev/com.ibm.ws.ejbcontainer.bindings_fat/test-applications/EJB3DefBndWeb.war/src/com/ibm/ws/ejbcontainer/bindings/defbnd/web/DefaultJavaColonBindingsServlet.java [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+ */
+@SuppressWarnings("serial")
+@WebServlet("/DefaultJavaColonBindingsServlet")
+public class DefaultJavaColonBindingsServlet extends FATServlet {
+
+    /**
+     * lookup and call local business interface using java: global.
+     */
+    @Test
+    public void testJavaColonLocalBI() throws Exception {
```
Comment (brideck) [inline dev/com.ibm.ws.ejbcontainer.bindings_fat/test-applications/EJB3DefBndWeb.war/src/com/ibm/ws/ejbcontainer/bindings/defbnd/web/DefaultJavaColonBindingsServlet.java]: I know you're just porting a legacy test here, but we probably shouldn't be using remote lookups, etc. for these two local tests now that this is going to open source.  Do we have a lookup method that doesn't do a narrow that we could use?  The assertion text should also be updated to reflect the correct interface names.

Comment (olendvcook) [reply dev/com.ibm.ws.ejbcontainer.bindings_fat/test-applications/EJB3DefBndWeb.war/src/com/ibm/ws/ejbcontainer/bindings/defbnd/web/DefaultJavaColonBindingsServlet.java]: @brideck I've changed it, thanks for spotting.

**Comment by olendvcook** [comment]
#build

**Comment by olendvcook** [comment]
for #7974

**Pull request merged**